app/src/main/kotlin/com/wire/android/ui/home/conversationslist/CallsScreen.kt

```diff
@@ -22,6 +22,7 @@ fun CallsScreen(
     onEditConversationItem: (ConversationItem) -> Unit,
     onScrollPositionChanged: (Int) -> Unit = {},
     onOpenUserProfile: (UserId) -> Unit,
+    openConversationNotificationsSettings: (ConversationItem) -> Unit,
```

Comment:
maybe to follow the same pattern, we can rename it to
```suggestion
    onOpenConversationNotificationsSettings: (ConversationItem) -> Unit,
```

Reply:
done